docs/user-guide/ui.md

```diff
@@ -2,76 +2,68 @@
 assignees:
 - bryk
 - mikedanese
+- rf232
 
 ---
 
 
-Dashboard (the web-based user interface of Kubernetes) allows you to deploy containerized applications to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resources itself. You can use it for getting an overview of applications running on the cluster, as well as for creating or modifying individual Kubernetes resources and workloads, such as Daemon sets, Pet sets, Replica sets, Jobs, Replication controllers and corresponding Services, or Pods.
+Dashboard (the web-based user interface of Kubernetes) allows you to deploy containerized applications to a Kubernetes cluster, troubleshoot them, and manage the cluster and its resources itself. You can use it for getting an overview of applications running on the cluster, as well as for creating or modifying individual Kubernetes resources and workloads, such as Daemon Sets, Stateful Sets, Replica Sets, Jobs and corresponding Services, or Pods.
 
-Dashboard also provides information on the state of Pods, Replication controllers, etc. and on any errors that might have occurred. You can inspect and manage the Kubernetes resources, as well as your deployed containerized applications. You can also change the number of replicated Pods, delete Pods, and deploy new applications using a deploy wizard.
+Dashboard also provides information on the state of Deployments, Replica Sets, etc. and on any errors that might have occurred. You can inspect and manage the Kubernetes resources and applications deployed on the cluster. For example, you can scale a Deployment, initiate a rolling update, restart a pod or deploy new applications using a deploy wizard.
```

Comment:
"Dashboard also provides information on the state of Kubernetes resources in your cluster, and on any errors that may have occurred."

The second sentence is unnecessary (repeats information from the first paragraph, but slightly reworded).

The last sentence needs to move to the end of the previous paragraph.